tool/src/cpp2/ty.rs

```diff
@@ -70,6 +71,20 @@ struct NamedType<'a> {
     type_name: Cow<'a, str>,
 }
 
+/// Everything needed for rendering a method
+struct MethodInfo<'a> {
+    method: &'a hir::Method,
+    return_ty: Cow<'a, str>,
+    type_name: Cow<'a, str>,
+    method_name: Cow<'a, str>,
+    pre_qualifiers: Vec<Cow<'a, str>>,
```

Comment:
nit: docs for the non-obvious ones, please

Reply:
Done